app/components/page-components/Chatbot/shared/ChatbotAliases.vue.ts

```diff
@@ -1,7 +1,7 @@
 import { Component, Prop } from 'vue-property-decorator';
 import ChatbotBase from 'components/page-components/Chatbot/ChatbotBase.vue';
 
-import { ITextMetadata } from 'components/shared/inputs/index';
+import { ITextMetadata } from '../../../shared/inputs';
```

Comment:
i think for a lot of these we prefer the aliased `components/shared..` convention, it's a lot clearer to see where files are coming from that way